FirebasePerformance/Sources/FPRClient.m

```diff
@@ -127,7 +125,7 @@ - (BOOL)startWithConfiguration:(FPRConfiguration *)config error:(NSError *__auto
 
 #ifdef TARGET_HAS_MOBILE_CONNECTIVITY
     // Create telephony network information object ahead of time to avoid runtime delays.
-    FPRNetworkInfo();
+    NetworkInfo();
```

Comment:
The prefix should stay on this symbol.  Otherwise there is the potential for build failures for any client apps that have a `NetworkInfo` function in their app or other library.

Reply:
Thanks Paul! I delete it due to symbol collision in FPRProtoUtils files. I have commented FPRProtoUtils files and restored the prefix.